promscale/installation/upgrade.md

```diff
@@ -0,0 +1,70 @@
+# Upgrade Promscale
+Promscale consists of the Promscale Connector, and the Promscale PostgreSQL
+extension. When you upgrade your Promscale installation, you need to check both
+components. This section contains important information about upgrading.
+
+The process for updating your Promscale installation is:
+
+1.  Check that the most recent required version of the Promscale extension is
+    available.
+1.  Stop all Promscale Connector instances connected to the database.
+1.  Perform the upgrade.
+1.  Start the new version of the Promscale Connector, and wait for it to begin
```

Comment:
```suggestion
1.  Stop all Promscale Connector instances connected to the database.
1.  Start the new version of the Promscale Connector, and wait for it to begin
```

Reply:
There is no upgrade to be performed since the upgrade happens automatically once the Promscale Connector is installed.